### ISSUE TITLE
exFAT support in initrd

### ISSUE THREAD

**Issue by Mexit** [Nitrux/nitrux-bug-tracker#26]
**Describe the bug**

Nitrux fails to boot if the ISO is on exFAT filesystem. 

**To Reproduce**

Boot ISO from grub2 on exFAT filesystem.

**Expected behavior**

System start-up.

**Screenshots**
```
BusyBox v1.30.1 (Ubuntu 1:1.30.1-6ubuntu2) built-in shell (ash)
Enter 'help' for a list of built-in commands.

(initramfs) 
Could not find the ISO /ISOs/nitrux-release-amd64_2021.05.10.iso
This could also happen if the file system is not clean because of an operating
system crash, an interrupted boot process, an improper shutdown, or unplugging
of a removable device without first unmounting or ejecting it.  To fix this,
simply reboot into Windows, let it fully start, log in, run 'chkdsk /r', then
gracefully shut down and reboot back into Windows. After this you should be
able to reboot again and resume the installation.

```

**Desktop (please complete the following information):**
- nitrux-release-amd64_2021.05.01.iso
- nitrux-release-minimal-amd64_2021.05.01.iso

**Additional context**

The exFAT module is missing in initrd.

**Comment by UriHerrera** [comment]
I've added an [exfat module to the initram of the ISO](https://github.com/Nitrux/iso-tool/commit/8d8be6c9dd639e092a2447d55dd24797e4325c45). Please test the recent ISOs.

**Comment by Mexit** [comment]
Unfortunately I don't see the exfat module in initrd (nitrux-nx-desktop-plasma-99bbba9e-amd64.iso):

```
$ cat lib/modules/6.3.4-1-liquorix-amd64/modules.order | grep exfat
kernel/fs/exfat/exfat.ko
```

```
$ tree lib/modules/6.3.4-1-liquorix-amd64/kernel/fs/
lib/modules/6.3.4-1-liquorix-amd64/kernel/fs/
├── btrfs
│   └── btrfs.ko
├── cifs
│   └── cifs.ko
├── ext4
│   └── ext4.ko
├── f2fs
│   └── f2fs.ko
├── fat
│   ├── fat.ko
│   └── vfat.ko
├── fscache
│   └── fscache.ko
├── fuse
│   └── fuse.ko
├── isofs
│   └── isofs.ko
├── jbd2
│   └── jbd2.ko
├── jfs
│   └── jfs.ko
├── lockd
│   └── lockd.ko
├── mbcache.ko
├── netfs
│   └── netfs.ko
├── nfs
│   ├── nfs.ko
│   ├── nfsv2.ko
│   ├── nfsv3.ko
│   └── nfsv4.ko
├── nfs_common
│   ├── grace.ko
│   └── nfs_acl.ko
├── nls
│   ├── nls_cp437.ko
│   ├── nls_iso8859-1.ko
│   └── nls_utf8.ko
├── overlayfs
│   └── overlay.ko
├── reiserfs
│   └── reiserfs.ko
├── smbfs_common
│   ├── cifs_arc4.ko
│   └── cifs_md4.ko
├── squashfs
│   └── squashfs.ko
├── udf
│   └── udf.ko
└── xfs
    └── xfs.ko

21 directories, 30 files
```

**Comment by UriHerrera** [comment]
![Screenshot_20230530_002622-1](https://github.com/Nitrux/nitrux-bug-tracker/assets/3053525/276da45e-7616-47bd-8a4b-a67e00ca6877)


I've generated a newer ISO, now the module file is listed with `lsinitramfs`. This change will be included in the next release.

**Comment by UriHerrera** [comment]
Added since May 29th, https://github.com/Nitrux/iso-tool/commit/edc202e0ac30d956d5b38af2b13414889139b4d7 and available since Nitrux 2.9.0.
